frontend/packages/operator-lifecycle-manager/src/utils.ts

```diff
@@ -0,0 +1,34 @@
+import * as _ from 'lodash';
+import { ObjectMetadata } from '@console/internal/module/k8s';
+import { ClusterServiceVersionKind } from './types';
+import { referenceForProvidedAPI } from './components';
+
+export const getInternalObjects = (annotations: ObjectMetadata['annotations']): string[] => {
```

Comment:
This is a nit, but I think it's a better abstraction if this takes in the CSV obj instead of the annotations.